docs/servers/add-gcp-server.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Add a Server to Google Cloud Platform (GCP)
+intro: Devopness allows you to provision a server on Google Cloud Platform using
+Google Compute Engine and manage it through Devopness.
+links:
+    overview:
+    quickstart:
+    previous:
+    next:
+    guides:
+    related:
+    featured:
+---
+
+# Add a Server to Google Cloud Platform (GCP)
+1. On Devopness, navigate to a project then select an environment
+1. If there's no other server in the selected environment, just click Add Server
+1. If a list of resource cards is displayed, find the Servers card and click View on it
+1. On the upper-right corner of the list click `ADD SERVER`
+1. Choose `Google Cloud Platform` as cloud provider
+1. Select a GCP credential using the drop-down menu
+    > If you dont have any GCP credentials listed, follow the [Add a Cloud Provider Credential](../cloud-provider-credentials/get-cloud-providers-credentails.md) guide before next step
+1. Click `NEXT`
+1. Select instance information:
+    - Select an avaliable region
+    - Select an avaliable instance type
+    - Fill the `Disk size` field with a value above 10GB
```

Comment:
See the documentation guidelines about how to keep docs short and straight to the point. Validation rules should not be a responsibility of the documentation: https://github.com/devopness/devopness/blob/main/docs/README.md?plain=1#L19 

```suggestion
    - Fill the `Disk size` field
```

Reply:
fixed